faucet/faucet.py

```diff
@@ -101,6 +102,7 @@ class Faucet(RyuAppBase):
     metrics = None
     notifier = None
     valves_manager = None
+    event_socket_heartbeat_time = 0 #seconds
```

Comment:
space after # 
Also kinda a difficult place for a comment b/c the chance that somebody is looking here at this point for the meaning  is small. 

Better to add an entry into docs/configuration.rst